frontend/src/components/Cell.tsx

```diff
@@ -28,18 +28,23 @@ export const Cell = ({
         on {`${date}`}
       </label>
       <input
-        type="number"
+        type="text"
         id={`${topic.issue.id}${topic.activity.id}${formatDate(
           date,
           "yyyy-MM-dd"
         )}`}
-        min={0}
         onChange={(event: any) => {
+          //makes sure that users can only input positive numbers up to 999.99999999...
+          //with an unlimited number of decimals behind the delimiter
+          event.target.value = event.target.value
+            .replace(/[^0-9.]/g, "")
+            .replace(/(^\d{3})\d/g, "$1")
```

Comment:
What about setting a limit on the numbers after the dot? At the moment I can enter as many as i want

Reply:
That would probably look like

```
.replace(/(\.\d{2}).*$/, "$1")
```

That is, remove everything after the second decimal digit (which seems like a reasonable number of decimal places).

Reply:
This is good, I used it now.